packages/sitecore-jss-nextjs/src/services/graphql-robots-service.ts

```diff
@@ -0,0 +1,89 @@
+import {
```

Comment:
Please move this service to `packages/sitecore-jss`. There's nothing nextjs-specific about this service (could technically be used in any framework). 

We have things organized by sub-modules (e.g. layout, i18n), and we should probably create one for this and upcoming SXA Headless work (redirects, sitemap.xml). 
Maybe call it `site` (since this is all based on `site` query / site settings)? @illiakovalenko @sc-addypathania what do you think?

You can then add pass-through exports to the framework SDKs via the root [`index.ts`](https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-nextjs/src/index.ts).

Reply:
@ambrauer I agree. We should move this to its own submodule and directly import it from there.